### PR TITLE
Update outdated message codes in Admin Console translations

### LINKED ISSUE #22300: Outdated message codes in Admin Console translations
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

admin/ui

### Describe the bug

When trying to use the web based translation system Weblate, I found that several old keys appeared in the translation files, while the original keys have been renamed. This fixes those entries. 

### Version

main

### Expected behavior

Weblate shouldn't drop the old translations just because the translation key has changed.

### Actual behavior

Weblate would drop those translations, because they are no longer connected to an original key.

### How to Reproduce?

Review the translation files

### Anything else?

I'll prepare a PR for this.

### PR DESCRIPTION
**Pull request by ahus1**
Closes #22300

### CODE REVIEW

**Comment by ahus1** [comment]
@jonkoops - here's another PR preparing things for Weblate. I'd be happy if you could review it. Thanks!

File: js/apps/admin-ui/public/locales/ja/authentication.json [hunk 1]
```diff
@@ -31,9 +31,9 @@
   "webAuthnPolicyAcceptableAaguids": "許容可能なAAGUID",
   "default": "DEFAULT",
   "flow": {
-    "browserFlow": "ブラウザーフロー",
-    "registrationFlow": "登録フロー",
-    "directGrantFlow": "ダイレクト・グラント・フロー"
+    "browser": "ブラウザーフロー",
+    "registration": "登録フロー",
+    "direct grant": "ダイレクト・グラント・フロー"
```
Comment (agagancarczyk) [inline js/apps/admin-ui/public/locales/ja/authentication.json:36]: this is great, we should also add these but translated ` "reset credentials": "Reset credentials flow",
    "clients": "Client authentication flow",
    "docker auth": "Docker authentication flow"`

File: js/apps/admin-ui/public/locales/lt/authentication.json [hunk 2]
```diff
@@ -18,9 +18,9 @@
     "none": "jokio"
   },
   "flow": {
-    "browserFlow": "Autentifikacijos seka",
-    "registrationFlow": "Registracijos seka",
-    "directGrantFlow": "Tiesioginių teisių seka"
+    "browser": "Autentifikacijos seka",
+    "registration": "Registracijos seka",
+    "direct grant": "Tiesioginių teisių seka"
```
Comment (agagancarczyk) [inline js/apps/admin-ui/public/locales/lt/authentication.json:23]: we should also add these but translated  "reset credentials": "Reset credentials flow", "clients": "Client authentication flow", "docker auth": "Docker authentication flow"

File: js/apps/admin-ui/public/locales/no/authentication.json [hunk 3]
```diff
@@ -18,9 +18,9 @@
     "none": "Ingen"
   },
   "flow": {
-    "browserFlow": "Nettleserflyt",
-    "registrationFlow": "Registreringsflyt",
-    "directGrantFlow": "Direct Grant Flyt"
+    "browser": "Nettleserflyt",
+    "registration": "Registreringsflyt",
+    "direct grant": "Direct Grant Flyt"
```
Comment (agagancarczyk) [inline js/apps/admin-ui/public/locales/no/authentication.json:23]: we should also add these but translated  "reset credentials": "Reset credentials flow", "clients": "Client authentication flow", "docker auth": "Docker authentication flow"

File: js/apps/admin-ui/public/locales/pt-BR/authentication.json [hunk 4]
```diff
@@ -18,9 +18,9 @@
     "none": "Nenhum"
   },
   "flow": {
-    "browserFlow": "Fluxo de browser",
-    "registrationFlow": "Fluxo de registro",
-    "directGrantFlow": "Fluxo de Direct Grant"
+    "browser": "Fluxo de browser",
+    "registration": "Fluxo de registro",
+    "direct grant": "Fluxo de Direct Grant"
```
Comment (agagancarczyk) [inline js/apps/admin-ui/public/locales/pt-BR/authentication.json:23]: we should also add these but translated  "reset credentials": "Reset credentials flow", "clients": "Client authentication flow", "docker auth": "Docker authentication flow"

File: js/apps/admin-ui/public/locales/ru/authentication.json [hunk 5]
```diff
@@ -18,9 +18,9 @@
     "none": "нет"
   },
   "flow": {
-    "browserFlow": "Сценарий браузера",
-    "registrationFlow": "Сценарий регистрации",
-    "directGrantFlow": "Сценарий Direct Grant Flow"
+    "browser": "Сценарий браузера",
+    "registration": "Сценарий регистрации",
+    "direct grant": "Сценарий Direct Grant Flow"
```
Comment (agagancarczyk) [inline js/apps/admin-ui/public/locales/ru/authentication.json:23]: we should also add these but translated  "reset credentials": "Reset credentials flow", "clients": "Client authentication flow", "docker auth": "Docker authentication flow"

**Review by agagancarczyk — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/public/locales/ja/authentication.json (hunk 1 above)
Comment (ahus1) [reply js/apps/admin-ui/public/locales/ja/authentication.json:36]: @agagancarczyk - thank you for suggesting this, I would consider adding new keys out of scope for this: The scope of this PR is to update keys of already translated parts, so they are not lost. I would assume that the UI falls back on the original English values for those elements that don't have a translation.

Once we have Weblate in place, the translators will see a list of all untranslated strings, and can then provide a translation. I suppose this comment would be true for all other languages where you added a similar comment.

Comment (agagancarczyk) [reply js/apps/admin-ui/public/locales/ja/authentication.json:36]: @ahus1 Great to hear that Weblate will catch any untranslated strings as there will be a few. I'm going to approve your PR now. Thanks for fixing this!

**Review by agagancarczyk — APPROVED** [review]
LGTM!

**Comment by jonkoops** [comment]
@ahus1 I was on PTO, but this all looks good to me retroactively.

**Pull request merged**